mypy/semanal.py

```diff
@@ -2861,7 +2861,7 @@ def visit_if_stmt(self, s: IfStmt) -> None:
     def visit_try_stmt(self, s: TryStmt) -> None:
         self.analyze_try_stmt(s, self)
 
-    def analyze_try_stmt(self, s: TryStmt, visitor: NodeVisitor,
+    def analyze_try_stmt(self, s: TryStmt, visitor: NodeVisitor[T],
```

Comment:
I think it does not make a difference in this case.

`NodeVisitor[None]` seems to be more readable though, so I'll change it to that.